packages/editor/src/client/panel/publishPagePanel.tsx

```diff
@@ -44,65 +45,16 @@ export function PublishPagePanel({
             description={t('pageEditor.panels.pagePending', {pendingPublishDate})}
           />
         )}
-        <Form fluid={true}>
-          <FormGroup>
-            <ControlLabel>{t('articleEditor.panels.publishTime')}</ControlLabel>
-            <DatePicker
-              style={{width: 100, marginRight: 8}}
-              placement="auto"
-              value={publishDate}
-              format="HH:mm"
-              ranges={[
-                {
-                  label: t('pageEditor.panels.now'),
-                  value: new Date()
-                }
-              ]}
-              onChange={publishDate => setPublishDate(publishDate)}
-            />
-            <DatePicker
-              style={{width: 130}}
-              placement="auto"
-              value={publishDate}
-              format="DD MMM YY"
-              ranges={[
-                {
-                  label: t('pageEditor.panels.now'),
-                  value: new Date()
-                }
-              ]}
-              onChange={publishDate => setPublishDate(publishDate)}
-            />
-          </FormGroup>
-          <FormGroup>
-            <ControlLabel>{t('articleEditor.panels.updateTime')}</ControlLabel>
-            <DatePicker
-              style={{width: 100, marginRight: 8}}
-              value={updateDate}
-              format="HH:mm"
-              ranges={[
-                {
-                  label: t('pageEditor.panels.now'),
-                  value: new Date()
-                }
-              ]}
-              onChange={updateDate => setUpdateDate(updateDate)}
-            />
-            <DatePicker
-              style={{width: 130}}
-              placement="auto"
-              value={updateDate}
-              format="DD MMM YY"
-              ranges={[
-                {
-                  label: t('pageEditor.panels.now'),
-                  value: new Date()
-                }
-              ]}
-              onChange={updateDate => setUpdateDate(updateDate)}
-            />
-          </FormGroup>
-        </Form>
+        <DateTimePicker
+          dateTime={publishDate ?? now}
+          label={t('articleEditor.panels.publishDate')}
+          changeDate={(date: Date) => setPublishDate(date)}
+        />
+        <DateTimePicker
+          dateTime={updateDate ?? now}
+          label={t('articleEditor.panels.updateDate')}
+          changeDate={date => setUpdateDate(date)}
```

Comment:
if you made `publishDate` mandatory above you should not need to type Date here.